addons/mail/views/mail_template_views.xml

```diff
@@ -23,71 +23,68 @@
+                                </div>
+                                <div class="oe_title"><label for="body_html"/></div>
+                                <div>
+                                    <field name="body_html" widget="html"
+                                            class="oe-bordered-editor"
+                                            options="{'style-inline': true, 'codeview': true }"/>
                                 </div>
-                                <field name="body_html" widget="html" class="oe-bordered-editor" options="{'style-inline': true, 'codeview': true }"/>
                                 <field name="attachment_ids" widget="many2many_binary"/>
                             </page>
-                            <page string="Email Configuration" name="email_configuration">
-                                <group>
-                                    <field name="email_from"
-                                            placeholder="Override author's email"/>
-                                    <field name="use_default_to"/>
-                                    <field name="email_to" attrs="{'invisible': [('use_default_to', '=', True)]}"
-                                            placeholder="Comma-separated recipient addresses"/>
-                                    <field name="partner_to" attrs="{'invisible': [('use_default_to', '=', True)]}"
-                                            placeholder="Comma-separated ids of recipient partners"/>
-                                    <field name="email_cc" attrs="{'invisible': [('use_default_to', '=', True)]}"
-                                            placeholder="Comma-separated carbon copy recipients addresses"/>
-                                    <field name="reply_to"
-                                            placeholder="Email address to which replies will be redirected when sending emails in mass"/>
-                                    <field name="scheduled_date" string="Scheduled Send Date"/>
-                                </group>
-                            </page>
                             <page string="Advanced Settings" name="advanced_settings">
-                                <group>
-                                    <field name="lang" placeholder="{{ object.partner_id.lang }}"/>
-                                    <field name="mail_server_id"/>
-                                    <field name="auto_delete"/>
-                                    <field name="report_template" domain="[('model','=',model)]"/>
-                                    <field name="report_name" attrs="{'invisible':[('report_template','=',False)]}"/>
-                                </group>
-                            </page>
-                            <page string="Dynamic Placeholder Generator" name="dynamic_placeholder_generator">
```

Comment:
I don't think we should remove that part until the editor cleanly supports the dynamic building generation from commands or its UI. Is it supported ?